tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
```

Comment:
I think we need a separate `tsconfig.json` and `tsconfig.build.json` at the top level, with the top level `tsconfig.json` extending `tsconfig.build.json` and defining the `paths` property:

```json
    "paths": {
      "@lest/*": [
        "packages/*/src"
      ]
    },
```